e2e/support/cypress-snapshots.config.js

```diff
@@ -2,4 +2,4 @@ const { defineConfig } = require("cypress");
 
 const { snapshotsConfig } = require("./config");
 
-module.exports = defineConfig({ e2e: snapshotsConfig });
+module.exports = defineConfig({ e2e: { ...snapshotsConfig, video: false } });
```

Comment:
No biggie, but there is a `snapshotsConfig` variable in the config.js where you could've set the `video` to false instead of doing it here in an export.

```js
const snapshotsConfig = {
  ...defaultConfig,
  video: false,
  specPattern: "e2e/snapshot-creators/**/*.cy.snap.js",
};
```